ci-operator/step-registry/netobserv/perf-test/cleanup/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- memodi
+- jotak
+- jpinsonneau
```

Comment:
for these files its better if you do a symbolic link to one file so when you need to update the owners you don't need to update multiple places